src/ui/osx/TogglDesktop/test2/AppIconFactory.m

```diff
@@ -20,7 +20,7 @@ + (NSImage *)appIconWithType:(AppIconType)type
 			icon = [NSImage imageNamed:@"AppIconActive"];
 			break;
 		case AppIconTypeDefault :
-			icon = [NSImage imageNamed:NSImageNameApplicationIcon];
+			icon = [NSImage imageNamed:@"AppIcon"];
```

Comment:
Fix the Missing Toggl Desktop Icon in the NSAlert.

Reply:
Nice fix, this also fixes the icon in delete popup. I've noticed it many times but had forgotten to create an issue.